lib/Arch/Sleigh/PPCArch.cpp

```diff
@@ -28,7 +28,7 @@ static constexpr auto kPPCVLERegName = "VLEReg";
 
 SleighPPCDecoder::SleighPPCDecoder(const remill::Arch &arch)
     : SleighDecoder(
-          arch, "ppc_64_isa_vle_be.sla", "ppc_64.pspec",
+          arch, "ppc_32_e200_be.sla", "ppc_64.pspec",
```

Comment:
the slaspec uses ppc_32.pspec but the only different between them are
```
  <context_data>
    <context_set space="ram">
      <set name="linkreg" val="0"/>
    </context_set>
  </context_data>
```
but it's not clear what this is for or if why it's necessary for 32bit but not 64bit?